src/theme/shared/fonts.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+const fonts = {
+  family: {
+    makeItSans: 'Make It Sans',
```

Comment:
TO DO:
I still need to custom the fontFamily with a Global.

Reply:
As far as I checked the Make It Sans font is created by external. And I need to add the Global component -> https://chakra-ui.com/docs/styled-system/recipes/using-fonts#option-2-using-font-face
I was thinking to apply the second option

Reply:
I will work on the custom font separately.